web-app/js/portal/filter/GeometryFilterPanel.js

```diff
@@ -37,8 +52,15 @@ Portal.filter.GeometryFilterPanel = Ext.extend(Portal.filter.BaseFilterPanel, {
         this.add(this.spatialSubsetControlsPanel);
     },
 
-    isVisualised: function() {
-        return false;
+    getFilterData: function() {
+
+        return {
+            name: this.filter.getName(),
+            visualised: this.isVisualised(),
+            cql: this.getCQL(),
+            humanValue: this._getCQLHumanValue(),
+            type: "geom"
```

Comment:
For reference: https://github.com/aodn/aodn-portal/blob/3bca2fab04bd8b241a2d09f9039898b3848e3c4f/web-app/js/portal/prototypes/OpenLayers.js#L181